HISTORY.md

```diff
@@ -1,12 +1,13 @@
-## v2.5.0b1 (2023-11-09)
+## v2.5.0 (2023-11-13)
 
-[GitHub release](https://github.com/pydantic/pydantic/releases/tag/v2.5.0b1)
+[GitHub release](https://github.com/pydantic/pydantic/releases/tag/v2.5.0)
```

Comment:
I think we should add a note saying something like "The code released in 2.5.0 is functionally identical to 2.5.0b1."